Mailbox/Person.cs

```diff
@@ -1,4 +1,41 @@
-﻿namespace Mailbox
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Mailbox
 {
+    public struct Person : IEquatable<Person>
+    {
+        string _FirstName;
+        string _LastName;
```

Comment:
```suggestion
        private string _LastName;
```

Should these be private?